### ISSUE TITLE
Environment variables

### ISSUE THREAD

**Issue by rajasegar** [fukamachi/caveman#137]
How do I implement something like `.env` files to store sensitive information and pass it along to the web server as environment variables. In Node.js we have something like 
https://www.npmjs.com/package/dotenv

**Comment by vindarel** [comment]
I believe you found out? https://dev.to/rajasegar/how-to-use-environment-variables-with-cl-dotenv-in-a-common-lisp-web-app-5eb5

**Comment by rajasegar** [comment]
@vindarel Yes, kind of, if @fukamachi  doesn't have anything to add, we can close this issue.
